### DISCUSSION TITLE
Criminal Life

### DISCUSSION THREAD

**Discussion by Kumotta** [CubicRP/Cubic#11] [General]
Would love it if there was a way to get thommy guns from some sort of convoy of npc's that can be spawned by getting a number from anywhere in the city in which you pay 20k for the number or some number around 20k and once you call the number the convoy of npc's spawn and you have to find them kill them break into the trunks with whatever item is needed and then you would receive 3 thommy guns each loaded with 45 bullets or so and these guns are considered class 2 or illegal weapons so cops would be prone to arrest you by any means or maybe even use deadly force.

**Comment by dwreyyes** [comment]
From my understanding, staff have tossed around ideas as to how these items are going to be implemented and circulated through the economy. What I can tell you is that discussions with ideas similar to yours have also been had. We do like the idea of a controlled event hosted by staff, or at least overseen by staff, that will give players the ability to obtain certain things such as weapons, cars, etc. 

Again, middle stages of development, lots of things are being thrown at the wall right now, excited to see what sticks!



**edit** 

Not sure if a script exists where this can be done EXACTLY as you have put out however.. with the 20k payment, AIs running the convoy.. so on so forth.

**Comment by ChivSuroo** [comment]
Server-wide events will be implemented in a variety of methods to get stuff like as automobiles, weapons, compounds, and so on. Will be done in such creative methods for individuals who have earned and established a significant mark in the city.
